### PR TITLE
Add GHA prerelease workflows

### PR DESCRIPTION
**Pull request by iowillhoit**
[@W-13907706@](https://gus.my.salesforce.com/apex/ADM_WorkLocator?bugorworknumber=W-13907706)

### CODE REVIEW

File: .github/workflows/create-github-release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: create-github-release
```
Comment (EstebanRomero84) [inline .github/workflows/create-github-release.yml:1]: Hi @iowillhoit - is there any documentation that helps me understand the reason of these changes and how to use the new workflow?

Comment (iowillhoit) [reply .github/workflows/create-github-release.yml:1]: Hey @EstebanRomero84, I've got a PR for docs open here https://github.com/salesforcecli/github-workflows/pull/63/files

Everything will work as it used to, the prerelease support is much more flexible with this change

File: .github/workflows/onRelease.yml [hunk 2]
```diff
@@ -22,11 +21,13 @@ jobs:
           ref: ${{ github.event.release.tag_name || inputs.tag  }}
       - uses: salesforcecli/github-workflows/.github/actions/getPreReleaseTag@main
         id: distTag
-
   npm:
     uses: salesforcecli/github-workflows/.github/workflows/npmPublish.yml@main
     needs: [getDistTag]
     with:
+      ctc: true
+      sign: true
```
Comment (iowillhoit) [inline .github/workflows/onRelease.yml:29]: @EstebanRomero84, I also turned on CTC and signing for your plugin. This should not cause any issues. Reach out if you see anything odd.

**Review by EstebanRomero84 — APPROVED** [review]
Thanks for the comments.

**Pull request merged**